Ginger/Ginger/AutomatePageLib/AddActionMenu/WindowExplorer/Common/WindowExplorerPage.xaml.cs

```diff
@@ -1435,7 +1435,7 @@ private async Task RefreshGridContent()
 
         private void xMainGrid_SizeChanged(object sender, SizeChangedEventArgs e)
         {
-            if (ActualWidth > 700)
+            if (ActualWidth > 1400)
```

Comment:
Please test this change wherever this windows is used. Also change target branch to beta